pkg/daemon/ceph/osd/volume.go

```diff
@@ -727,7 +727,7 @@ func GetCephVolumeRawOSDs(context *clusterd.Context, clusterInfo *client.Cluster
 		args = []string{cvMode, "list", "--format", "json"}
 	}
 
-	result, err := callCephVolume(context, args...)
+	result, err := callCephVolume(context, "list", args...)
```

Comment:
Isn't `"list"` already in the `args`? Oh wait, it's just confusing that it's used to determine which execute method to use. Separate comment added below.